api/src/middleware/assessmentsDummyRouter.ts

```diff
@@ -0,0 +1,291 @@
+import { Router } from 'express';
+
+import { itemEnvelope } from './responseEnvelope';
+import { BadRequestError } from './httpErrors';
+import {
+  insertToProgramParticipants,
+  insertToAssessmentSubmissions,
+} from '../services/assessmentsDummyService';
+const assessmentsDummyRouter = Router();
+
+interface ProgramParticipantsRow {
```

Comment:
Since we're not using templating in this file, go ahead and just collapse this interface into the actual code; that is to say, replace the name `ProgramParticipantsRow` on line 44 with what's between these curly braces. There's no real need to separate out the type declaration in this circumstance.